syntax = "proto3";

option go_package = "jd.com/jd-infer/xllm;xllm";
package xllm.proto;
option cc_enable_arenas = true;
option cc_generic_services = true;

import "common.proto";

message SessionInfo {
  string addr = 1;
}

service MooncakeTransferEngineService {
  rpc OpenSession(SessionInfo) returns (Status) {}
  rpc CloseSession(SessionInfo) returns (Status) {}
}
